bin/rcrestore

```diff
@@ -0,0 +1,58 @@
+#! /bin/bash
+
+if [[ ! -f $1 ]]
+then
+    echo "Usage: snap run rocketchat-server.rcrestore $SNAP_USER_COMMON/backup_file.tgz"
+    exit
+fi
+
+cd ${1%/*}
+if [[ -z $(pwd | grep "$SNAP_USER_COMMON") ]]
```

Comment:
I agree it looks weird and there might be a better solution. The reasoning was:  
In case they specify a file completely outside of `$SNAP_USER_COMMON`, like `snap run rocketchat-snap.restoredb /home/user/mybackup.tgz`, the snap wouldn't have access to the file. So to check, I'd navigate to that directory first, and then check if I ended up at `$SNAP_USER_COMMON`.

Secondly, checking that the file is directly under `$SNAP_USER_COMMON` may not be enough either, because it could be in a subdirectory like `$SNAP_USER_COMMON/backups`, for instance.